pkg/lib/bundlev2/nonInferrableCsv.go

```diff
@@ -0,0 +1,32 @@
+package bundlev2
```

Comment:
nit: I think the filename should use underscores instead of camelCase to separate words, ie `non_inferrable_csv.go` or something simpler like `csv_v2.go`